### ISSUE TITLE
PMD produces JUnit 4 errors for JUnit 5 tests

### ISSUE THREAD

**Issue by AdamCarroll** [VocabHunter/VocabHunter#34]
PMD produces errors such as the following for the JUnit 5 tests:
> JUnit 4 tests that set up tests should use the @Before annotation

To workaround this, the rules are [explicitly excluded in the PMD config](https://github.com/VocabHunter/VocabHunter/blob/d2f2e536ab0aec1a17063c8dde1365006d39706e/config/pmd/ruleset.xml#L15-L17):
````
<exclude name="JUnit4TestShouldUseBeforeAnnotation"/>
<exclude name="JUnit4TestShouldUseAfterAnnotation"/>
<exclude name="JUnit4TestShouldUseTestAnnotation"/>
````

**Comment by AdamCarroll** [comment]
Raised on the PMD project as pmd/pmd#940

**Comment by AdamCarroll** [comment]
This is fixed in 8a980fd3fe87b4c885cba39a3843029868e388d2 with the upgrade to PMD v6.7.0.
